### ISSUE TITLE
mpd only listening to ipv6

### ISSUE THREAD

**Issue by maweki** [mopidy/mopidy#1580]
I am running Mopidy 2.0.1 with DietPi on a NanoPi NEO.

I have the following settings for the mpd and web:

```
[mpd]
enabled = true
hostname = 0.0.0.0
port = 6600
password =

[http]
enabled = true
hostname = 0.0.0.0
port = 6680
```

But looking at `netstat -l`, we see that mpd grabs an `ipv6` interface and is not available in the network. Even setting `127.0.0.1` as the `hostname`, it takes `*` on `ipv6`

```
tcp        0      0 *:6680                  *:*                     LISTEN     
...
tcp6       0      0 0.0.0.0:6600            [::]:*                  LISTEN     
```

**Comment by maweki** [comment]
Setting the hostname to `::` I get the following entry for port `6600` (and none in ipv4)

```
tcp6       0      0 [::]:6600               [::]:*                  LISTEN     
```

And for `127.0.0.1`, I get the following (none for ipv4 again)

```
tcp6       0      0 localhost:6600          [::]:*                  LISTEN     
```

**Comment by kingosticks** [comment]
Regardless of what netstat says (it's not a reliable tool when ipv6-ipv4 mapping is going on), is the mpd interface actually unavailable in all these cases?

**Comment by maweki** [comment]
Oh, you're right. `mpd` is actually available with `0.0.0.0` and `::`.

Then my problem with `upmpdcli` must be somewhere else. :(

Thank you

**Comment by kingosticks** [comment]
If you get stuck, try posting on discuss.mopidy.com as other people might be using upmpdcli there.
